app/src/main/java/com/soen390/conumap/ui/directions/DirectionsFragment.kt

```diff
@@ -57,7 +60,12 @@ class DirectionsFragment : Fragment() {
 
     override fun onActivityCreated(savedInstanceState: Bundle?) {
         super.onActivityCreated(savedInstanceState)
-        viewModel = ViewModelProviders.of(this).get(DirectionsViewModel::class.java)
+        //DELETE viewModel = ViewModelProviders.of(this).get(DirectionsViewModel::class.java)
```

Comment:
Is it possible to bring this back? The viewModel is necessary to communicate with the Activity and I think we should keep it for now

Reply:
This is because this viewModel is not used yet and it is better for us to only use the SearchBarViewModel to communicate the data between fragments more easily
[same comment as below since same problem]